tap_intercom/stream.py

```diff
@@ -16,25 +16,18 @@ def __init__(self, config: Dict):
     def do_sync(self, tap_stream_id: str, state: Optional[datetime] = None):
 
         prev_bookmark = None
-        start_date, end_date = self.__get_start_end(state)
-        with singer.metrics.record_counter(self.tap_stream_id) as counter:
-            with Transformer(
-                integer_datetime_fmt=UNIX_SECONDS_INTEGER_DATETIME_PARSING
-            ) as transformer:
-                try:
-                    data = self.intercom.get_records(self.tap_stream_id)
-                    for d, replication_value in data:
+        with singer.metrics.record_counter(tap_stream_id) as counter:
+
+            try:
+                data = self.intercom.get_records(tap_stream_id)
+                for record, replication_value in data:
 
                         if replication_value and (
                             start_date >= replication_value or end_date <= replication_value
                         ):
                             continue
 
-                        record = transformer.transform(d, self.schema, self.mdata)
```

Comment:
Why not use `transform`?

Reply:
https://www.notion.so/dreamdata/Law-of-the-Taps-abac8f608b61457dbe47b648a6216af3#33f24a6ee5f94b4382b97bfa12f6da11